comit/src/expiries.rs

```diff
@@ -4,7 +4,7 @@
 //! atomic. One expiry for the alpha ledger and one for the beta ledger. The
 //! beta ledger expiry time must not elapse before the alpha ledger expiry time.
 
-mod config;
+pub mod config;
```

Comment:
We are re-exporting it now, this should go away :)

Reply:
ooops, slipped in. thanks.